src/misc.c

```diff
@@ -776,8 +776,9 @@ void skelout (void)
 					outn ((char *) (yydmap_buf.elts));
 			}
             else if (cmd_match (CMD_DEFINE_YYTABLES)) {
-                out_str("#define YYTABLES_NAME \"%s\"\n",
-                        tablesname?tablesname:"yytables");
+		if(tablesext)
+                   out_str("#define YYTABLES_NAME \"%s\"\n",
+                           tablesname?tablesname:"yytables");
```

Comment:
Sorry to bother. But can you fix the coding style a little bit as you fix the code here?
There should be a space between `if` and left parenthesis (in order to distinguish the syntax of function or macro call).  
It's also better to put spaces before and after each of the `? :` operators since they have low precedence in the language.

Reply:
Done.